docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -90,10 +89,37 @@ To update the metadata and playback information on the server side, and to retri
         Tizen.Log.info(LogTag, $"Playback state is {controller.GetPlaybackState()}");
         ```
 
-<a name="send_media"></a>
+## Updating and Retrieving Playlist
+
+To create a playlist from the server side, create a new instance of the [Tizen.Multimedia.Remoting.MediaControlPlaylist](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlPlaylist.html) class:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavorite");
+    ```
+
+You can also create playlist with Metadata using the following code:
+
+    ```
+    var playlist = new MediaControlPlaylist("MyFavoriate",
+        new Dictionary<string, MediaControlMetadata>()
+        {
+            {"Song1", new MediaControlMetadata() {Author = "Someone"} },
+            {"Song2", new MediaControlMetadata()}
+        }
+    );
+    ```
+
+To retrieve the playlist and metadata information on the client side:
+    ```
+        controller.PlaylistUpdated += (s, e) =>
+        {
+            Tizen.Log.Info(LogTag, $"Updated mode : {e.Mode}, Name : {e.Name}, The title of first song : {e.Playlist.GetMetadata("IDX1").Title}");
+        };
+    ````
```

Comment:
@hsgwon  This code block and line is not rendering properly in the view. Please check the following screenshot regarding the same:
![image](https://user-images.githubusercontent.com/42992085/60589846-11d18d80-9db8-11e9-868e-cf687aae8a22.png)

Kindly remove the extra backtick (`) at the end of this code block. Use only three backticks (```) to start and end the code block. For more information regarding code blocks, please refer to: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#code

Reply:
It's typo and will be fixed.